src/IMUI/imguiWrapper.cc

```diff
@@ -121,10 +121,12 @@ imguiWrapper::setupFontTexture(const IMUISetup& setup) {
 
     io.Fonts->AddFontDefault();
     for (int i = 0; i < setup.numFonts; i++) {
-        ImFontConfig fontConfig;
-        fontConfig.FontDataOwnedByAtlas = false;
         const auto& desc = setup.fonts[i];
-        this->fonts[i] = io.Fonts->AddFontFromMemoryTTF(desc.ttf_data, desc.ttf_size, desc.font_height, &fontConfig);
+		desc.font_config->FontDataOwnedByAtlas = false;
+		if (desc.compressed)
```

Comment:
Please add curly braces in this if/else, otherwise it's looking good

Reply:
Okay great, I added a commit for this.